apps/studio/src/features/editing-experience/utils.ts

```diff
@@ -1 +1,24 @@
+import type { IsomerComponent, ProseProps } from "@opengovsg/isomer-components"
+import { getComponentSchema } from "@opengovsg/isomer-components"
+import Ajv from "ajv"
+
 export const dataAttr = (value: unknown) => (!!value ? true : undefined)
+
+const proseSchema = getComponentSchema("prose")
+
+export const inferAsProse = (component?: IsomerComponent): ProseProps => {
```

Comment:
is this used anywhere? if not can delete

Reply:
removed!